editors/code/src/commands.ts

```diff
@@ -185,15 +185,21 @@ export function parentModule(ctx: Ctx): Cmd {
 
 export function ssr(ctx: Ctx): Cmd {
     return async () => {
+        const editor = vscode.window.activeTextEditor;
         const client = ctx.client;
-        if (!client) return;
+        if (!editor || !client) return;
+
+        const position = editor.selection.active;
```

Comment:
I think this is OK as first solution, but we probably should move to that "ssr via assit" idea some time in the future. Currently, the association between position and action is pretty weak.

It might also be a good idea to think about how this should work in the CLI setting.